docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+.NET Core
+=========
+
+.NET Core is a general purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows, OS X and Linux and can be used in device, cloud and embedded/IoT scenarios. The supported Operating Systems (OS), CPUs and application scenarios will grow over time, provided by Microsoft, other companies and individuals.
```

Comment:
Link to reqs and OS matrix? Or embed it here? (I would link to avoid having to update all over the place)